nemo_curator/backends/xenna/executor.py

```diff
@@ -43,8 +43,14 @@ def __init__(self, config: dict[str, Any] | None = None):
                 - execution_mode: 'streaming' or 'batch' (default: 'streaming')
                 - cpu_allocation_percentage: CPU allocation ratio (default: 0.95)
                 - autoscale_interval_s: Auto-scaling interval (default: 180)
+            ignore_head_node (bool, optional): Whether to ignore the head node (default: False)
+                Not supported by XennaExecutor. Ignored if set to True.
         """
-        super().__init__(config)
+        super().__init__(config, ignore_head_node)
+        if self.ignore_head_node:
+            msg = "ignore_head_node is not supported by XennaExecutor. Use RayDataExecutor or RayActorPoolExecutor instead."
+            raise ValueError(msg)
```

Comment:
From the docstring, it seems like we should log a warning instead of raise an error?